src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) =>
+    String(a) > String(b)) {
```

Comment:
The default compare function String(a) > String(b) is not accurate, it would return a boolean instead of -1, 0, or 1, which is expected by the sort function. You should adjust this to function(a, b) { return String(a) > String(b) ? 1 : String(a) < String(b) ? -1 : 0; }